test/integration/aws/default/build/route_table.tf

```diff
@@ -23,10 +23,27 @@ resource "aws_route_table" "rtb" {
   }
 }
 
+resource "aws_route_table" "rtb2" {
+  vpc_id = "${data.aws_vpc.default.id}"
+
+  route {
+    cidr_block = "172.32.1.0/24"
+    gateway_id = "${data.aws_internet_gateway.default.id}"
+  }
+
+  tags {
```

Comment:
As we're not testing tags yet, I'd prefer that we remove tagging from the fixture.  Keep it minimal, meaningful properties only.